src/tests/mocks/account_mocks.cairo

```diff
@@ -0,0 +1,183 @@
+#[starknet::contract]
+mod DualCaseAccountMock {
+    use openzeppelin::account::Account as account_component;
+    use openzeppelin::introspection::src5::SRC5 as src5_component;
+
+    component!(path: account_component, storage: account, event: AccountEvent);
+    component!(path: src5_component, storage: src5, event: SRC5Event);
```

Comment:
Out account implements SRC6, but it also implements public key setter and getter, declarer, and deployable interfaces... Should we remove all that and call it SRC6 to later add the rest in a preset or the wizard? If not I lean towards keeping the name account_component, because we are not providing an SRC6 component, but our extension of it